src/ast2ram/provenance/TranslationStrategy.cpp

```diff
@@ -0,0 +1,46 @@
+/*
+ * Souffle - A Datalog Compiler
+ * Copyright (c) 2020 The Souffle Developers. All rights reserved
+ * Licensed under the Universal Permissive License v 1.0 as shown at:
+ * - https://opensource.org/licenses/UPL
+ * - <souffle root>/licenses/SOUFFLE-UPL.txt
+ */
+
+/************************************************************************
+ *
+ * @file TranslationStrategy.cpp
+ *
+ ***********************************************************************/
+
+#include "ast2ram/provenance/TranslationStrategy.h"
+#include "ast2ram/provenance/ClauseTranslator.h"
+#include "ast2ram/provenance/UnitTranslator.h"
+#include "ast2ram/seminaive/ConstraintTranslator.h"
+#include "ast2ram/seminaive/ValueTranslator.h"
+#include "ast2ram/utility/TranslatorContext.h"
+#include "ram/Condition.h"
+#include "ram/Expression.h"
+#include "souffle/SymbolTable.h"
+
+namespace souffle::ast2ram::provenance {
+
+ast2ram::UnitTranslator* TranslationStrategy::createUnitTranslator() const {
+    return new UnitTranslator();
```

Comment:
Agree, will leave it for a later PR though since this is easier for now + code will move around. (These translator PRs are incremental to avoid huge breaking changes).